cfme/configure/settings.py

```diff
@@ -269,6 +269,34 @@ def timezone(self, value):
         if view.visualdisplay.time_zone.fill(value):
             view.save.click()
 
+    @property
+    def grid_view_read(self):
```

Comment:
**REQUIRED**
Bad naming is here. You named the property like a method. It should be something like `grid_view_entities`.